boost-maven/boost-boosters/booster-beanValidation20/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>boost.boosters</groupId>
+    <artifactId>beanValidation</artifactId>
+    <version>2.0-1.0-M1-SNAPSHOT</version>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>boost.boosters</groupId>
+                <artifactId>boost-ee8-apis-bom</artifactId>
+                <version>0.1-SNAPSHOT</version>
```

Comment:
We said we were going to move away from this right?   I think that's still in the backlog but you might as well start.